### ISSUE TITLE
Wrong Model Car

### ISSUE THREAD

**Issue by Phluphy** [itsMeDavidV/Watchla-Support#149]
I have a Model S. Main app screen says Model Y and when I tap vehicle on main app screen, the next page correctly says Model S where you can pick from multiple cars. 

![IMG_9149](https://github.com/itsMeDavidV/Watchla-Support/assets/141986645/82d28334-12d9-488a-864f-25fadf098ad6)
![IMG_9150](https://github.com/itsMeDavidV/Watchla-Support/assets/141986645/96d7cc23-c791-41c7-966b-6139221dd906)
![IMG_9151](https://github.com/itsMeDavidV/Watchla-Support/assets/141986645/0a557ae0-2633-4659-8810-cad5e550aa84)

**Comment by Framaldonado** [comment]
I also have this issue.

**Comment by itsMeDavidV** [comment]
Hi @Phluphy sorry for the late reply. That's very strange, is this behavior still occurring for you? If so, could you send over your Watchla Support code via https://watchla.app/contact

**Comment by Phluphy** [comment]
Done!
    On Thursday, September 14, 2023 at 05:17:52 PM EDT, David Villegas ***@***.***> wrote:  
 
 


Hi @Phluphy sorry for the late reply. That's very strange, is this behavior still occurring for you? If so, could you send over your Watchla Support code via https://watchla.app/contact

**Comment by Phluphy** [comment]
Hi. Any update? I’m still having this same issue.

**Comment by itsMeDavidV** [comment]
Yes, we think have honed in on the source of this issue, however Watchla is undergoing a large rebuild currently so all focus is on that right now. This issue should not occur after the rebuild. 

[You can find more info on the Watchla rebuild on X, as well as future updates on its timeline](https://x.com/WatchlaApp/status/1711926100736315744?s=20)

**Comment by Phluphy** [comment]
Thank you, David.  Watchla is a wonderful app and I'm happy to use it!  
I'm not on X so updates there are hard for me to obtain so can you provide an approximate ETA just I know I don't keep flooding your Inbox with questions?  I know the ETA is not guaranteed but it does give me an idea of how long to wait.
    On Sunday, October 15, 2023 at 12:12:05 PM EDT, David Villegas ***@***.***> wrote:  
 
 


Yes, we think have honed in on the source of this issue, however Watchla is undergoing a large rebuild currently so all focus is on that right now. This issue should not occur after the rebuild.

You can find more info on the Watchla rebuild on X, as well as future updates on its timeline

**Comment by itsMeDavidV** [comment]
@Phluphy thank you for your kind words! So the rebuild will consist of 2 parts. The first part should be released within a month at which time we can correct small issues like this, however this a very rough estimation.

**Comment by knovoselic** [comment]
Any updates on this? I have the same issue.
